apps/bridge/package.json

```diff
@@ -13,7 +13,7 @@
     "@bugsnag/plugin-react": "^7.19.0",
     "@eth-optimism/contracts-ts": "^0.16.2",
     "@eth-optimism/indexer-api": "^0.0.4",
-    "@headlessui/react": "latest",
+    "@headlessui/react": "^1.7.19",
```

Comment:
`latest` is now a major version up (2.x.x) and doesn't have the same api surface that `bridge` is expecting.